### ISSUE TITLE
Idea: edit/crop and quote images

### ISSUE THREAD

**Issue by Bondarz** [kriztan/Pix-Art-Messenger#428]
hello, i am new to XMPP and am currently missing two features

1. the possibility of cropping an image before sending it.
2. the selection of images/files to quote, just like text chats. 

Best regards and thanks

**Comment by Echolon** [comment]
Hi @Bondarz,

would be good if you could split your request into two separate issues. This will increase recognition from development.

**Comment by Bondarz** [comment]
Should i do it, or can you edit this post in "cropping an image" and then i start a new thread for quotes?

**Comment by Echolon** [comment]
@Bondarz Yes I do for you, but you can always edit **your** issues yourself!
